Formula/dcmtk.rb

```diff
@@ -17,20 +17,23 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
+  option "with-dicomdict", "Build with baked-in DICOM data dictionary."
+  option "without-openssl", "Build without support for OpenSSL."
```

Comment:
`doxygen` is a little different because all three current options were birthed out of previous options that were more explicit in what they did, but caused issues with Homebrew's dependency management/display/etc. It's likely that if `doxygen` would be written from scratch today the only option left standing as an explicit option text would be the `with-graphviz` one.

Simply changing the `depends_on "openssl"` to `depends_on "openssl" => :recommended` combined with your change inside the `def install` block here is sufficient to achieving what you wish to. You may wish to make it more explicit in case upstream changes the default, but ultimately that's up to @ilovezfs to make a final decision on.

```ruby
if build.with? "openssl"
  args << "-DDCMTK_WITH_OPENSSL=YES"
else
  args << "-DDCMTK_WITH_OPENSSL=NO"
end
```